src/pages/home/report/ReportActionItem.js

```diff
@@ -203,6 +204,22 @@ class ReportActionItem extends Component {
                     checkIfContextMenuActive={this.checkIfContextMenuActive}
                 />
             );
+        } else if (this.props.action.actionName === CONST.REPORT.ACTIONS.TYPE.TASKCOMPLETED) {
+            children = (
+                <TaskAction
+                    taskReportID={this.props.action.originalMessage.taskReportID.toString()}
+                    actionName={this.props.action.actionName}
+                    isHovered={hovered}
+                />
+            );
+        } else if (this.props.action.actionName === CONST.REPORT.ACTIONS.TYPE.TASKCANCELED) {
```

Comment:
We can just add another || for this.props.action.actionName === CONST.REPORT.ACTIONS.TYPE.TASKCANCELED since we're returning the same thing